### PR TITLE
Add pigpiod, systemd service, auto-download JAR

### PR DESCRIPTION
**Pull request by Bankst**

### CODE REVIEW

File: stage2/01-sys-tweaks/files/photonvision.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=PhotonVision server
+[Service]
+WorkingDirectory=/opt/photonvision
+ExecStart=java -jar /opt/photonvision/photonvision.jar
+ExecStop=/bin/systemctl kill photonvision
```
Comment (Modelmat) [inline stage2/01-sys-tweaks/files/photonvision.service:6]: Uh, is this necessary?
systemd would normally kill dependent processes anyhow.

Also, newlines between [Service] and [Install] would be nice :)

File: stage2/01-sys-tweaks/files/photonvision.service [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=PhotonVision server
```
Comment (Modelmat) [inline stage2/01-sys-tweaks/files/photonvision.service:2]: Probably would want to have 
`After=NetworkManager-wait-online`?

Comment (Bankst) [reply stage2/01-sys-tweaks/files/photonvision.service:2]: I thought about this but decided against it, as we take over network control anyways. Whether the interface comes up at boot or not, we want the service running to be ready for when it does.

File: stage2/01-sys-tweaks/files/photonvision.service (hunk 1 above)
Comment (Bankst) [reply stage2/01-sys-tweaks/files/photonvision.service]: resolved in c6c7a69

**Review by mcm001 — APPROVED** [review]

**Pull request merged**